compiler/compile/OMRCompilation.cpp

```diff
@@ -2173,6 +2173,86 @@ OMR::Compilation::isInlinedDirectCall(uint32_t index)
    return _inlinedCallSites[index].directCall();
    }
 
+/*
+ * Get the size of the OSRCallSiteRemat table, based on the number of pending push
```

Comment:
This sounds a bit confusing. Perhaps it should read: Get the number of pending push slots in the given caller. This is the number of entries in the CallSiteRemat entry for the caller - or something like that.